common/api/presentation-common.api.md

```diff
@@ -368,6 +368,9 @@ export enum ContentSpecificationTypes {
 // @public
 export type ContentUpdateInfo = typeof UPDATE_FULL;
 
+// @public
```

Comment:
should new apis be public from the get go or should it be public preview or beta?